_includes/gh-buttons.html

```diff
@@ -1,11 +1,11 @@
 <div class="gh-btns">
   <!-- Follow -->
-  <iframe src="http://ghbtns.com/github-btn.html?user=hilios&type=follow"
+  <iframe src="https://ghbtns.com/github-btn.html?user=hilios&type=follow"
```

Comment:
I don't think it is, actually. HTTPS is not considered to have sufficient performance penalties for it to be necessary to ever fall back to HTTP, and I don't really see any reason why anyone would prefer HTTP at this point, when the server provides it. See e.g. the note at the top of [this post by Paul Irish](http://www.paulirish.com/2010/the-protocol-relative-url/).